Formula/vsce.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Vsce < Formula
+  desc "Tool for packaging, publishing and managing VS Code extensions"
+  homepage "https://code.visualstudio.com/api/working-with-extensions/publishing-extension#vsce"
+  url "https://registry.npmjs.org/vsce/-/vsce-2.7.0.tgz"
+  sha256 "68016d50a3208624c525dd8d214ac81fb37772d94f1565740a2e9d061a25327c"
+  license "MIT"
+  head "https://github.com/microsoft/vscode-vsce.git", branch: "main"
+
+  livecheck do
+    url "https://registry.npmjs.org/vsce/latest"
+    regex(/["']version["']:\s*?["']([^"']+)["']/i)
+  end
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir[libexec/"bin/*"]
+  end
+
+  test do
+    assert_match version.to_s, shell_output(bin/"vsce --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.